CMakeLists.txt

```diff
@@ -7,12 +7,16 @@ if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
                   "${CMAKE_BINARY_DIR}/conan.cmake")
 endif()
 
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development.Module)
```

Comment:
Commit says "Find Python earlier", but not why. Also, this seems to be added and not moved, how did we find Python before?

Reply:
Pybind used to find it for us before. Explained more in the commit message. Let me know if I need to expand more.